### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by JoshLamptey**
but i didn't delete the contributing.md because i thought maybe you might wanna review the change and do it yourself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,89 @@
 # cdnExtension
```
Comment (RamGoel) [inline README.md]: Please update it to EasyCDN

File: README.md [hunk 2]
```diff
@@ -1,12 +1,89 @@
...
 
-No more Searching for CDN Links again and again on the Web. Our Tool cdnExtension solves this problem by keeping all the popular cdn links right on the top of your browser.
```
Comment (RamGoel) [inline README.md:3]: Please update here as well.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,89 @@
...
+easyCDN is a time-saving tool designed to simplify the process of accessing popular CDN (Content Delivery Network) links directly from your web browser. Say goodbye to the hassle of searching for CDN links on the web; easyCDN keeps all the popular CDN links conveniently at the top of your browser.
 
+## Table of contents
+ *[Features](#features)
+ *[Contributing](#contributing)
+ *[Contributors](#contributors)
```
Comment (RamGoel) [inline README.md:8]: Convert this to a list like -

- [Features](#features)
- [Contributing](#contributing)
- [Contributors](#contributors)

File: README.md [hunk 4]
```diff
@@ -1,12 +1,89 @@
...
+Now submit the pull request.
+
+
+
+
+Happy Contributing :)
+
 ## contributors
 [Ram Goel](https://github.com/RamGoel)
+[JoshLamptey](https://github.com/JoshLamptey)
```
Comment (RamGoel) [inline README.md]: Add other contributors

File: README.md (hunk 1 above)
Comment (JoshLamptey) [reply README.md]: Okay I'm on it

Comment (JoshLamptey) [reply README.md]: I have created the pull request I'm waiting for your approval

**Comment by RamGoel** [comment]
@JoshLamptey some changes are still pending. check comments.

**Comment by JoshLamptey** [comment]
Okay, I've got to go to work. By the time I'll get home, I'll have a complete list of the changes, and I'll work on them and make a pr tonight 
Does that sound good

**Comment by JoshLamptey** [comment]
@RamGoel is the contributors name the only edit you want  done to the commit or you want me to delete the contributions file too

**Comment by RamGoel** [comment]
> @RamGoel is the contributors name the only edit you want done to the commit or you want me to delete the contributions file too

- Update cdnExtension to easyCDN everywhere in file (Keep title as EasyCDN)
- Change List to a proper list according to given comment
- Add other 3 contributors as well in list. (Prefarably icons)

**Comment by JoshLamptey** [comment]
Can I have the name of the contributors please

**Comment by RamGoel** [comment]
Try to add their profile logos if possible
image

**Comment by JoshLamptey** [comment]
Got it you will have it by tonight

**Review by RamGoel — APPROVED** [review]

**Pull request merged**